tests/runner.py

```diff
@@ -135,6 +138,12 @@ def decorated(f):
   return decorated
 
 
+def only_wasm_backend(note=''):
```

Comment:
How about `no_fastcomp` to match the existing `no_wasm_backend`?

Reply:
Sounds good, changing to `no_fastcomp`.